event_session_registration_multi_qty/models/event_session.py

```diff
@@ -1,20 +1,23 @@
-# -*- coding: utf-8 -*-
-# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
-
-from odoo import _, api, models
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
+from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError
 
 
 class EventSession(models.Model):
     _inherit = 'event.session'
 
+    registration_multi_qty = fields.Boolean(
```

Comment:
Why adding this related?

Reply:
Sure, it's useless. There was a view that had a reference to this field, although it wasn't declared, so...